src/NFSFile.cpp

```diff
@@ -673,8 +673,8 @@ bool CNFSFile::GetServerList(std::vector<kodi::vfs::CDirEntry>& items)
     std::string path(std::string("nfs://") + currentExport);
     if (path[path.size()-1] != '/')
       path += '/';
-    pItem.SetPath(strdup(path.c_str()));
-    pItem.SetLabel(strdup(currentExport.c_str()));
+    pItem.SetPath(path.c_str());
+    pItem.SetLabel(currentExport.c_str());
```

Comment:
All of these .c_str() can be dropped. Also there's some more lines that had me worried for memory safety, I'll check this out later.